router.go

```diff
@@ -177,5 +177,6 @@ func pathParms(swaggerUrl string) (params []string) {
 }
 
 // SwaggerUiTemplate contains the html for swagger UI.
+//
```

Comment:
👀

Reply:
Go FMT really wants that to go in, I tried to remove it before I hit you up for approval and it just reformatted it on save. I can probably force remove it.

Reply:
Remove it.